edb/common/debug.py

```diff
@@ -123,6 +123,9 @@ class flags(metaclass=FlagsMeta):
     log_metrics = Flag(
         doc="Log verbose statistics on connections and compiler behavior.")
 
+    migrations_via_ddl = Flag(
+        doc="Always use generated DDL when running migrations.")
```

Comment:
use -> apply or execute?

Reply:
No, "use" is more appropriate, because currently we generate the DDL, but use the source delta tree instead.  Regardless, this is a temporary flag until we fix all current testsuite failures and enable this behavior by default.